exercises/two-bucket/two_bucket.py

```diff
@@ -0,0 +1,2 @@
+def two_bucket(bucket_one, bucket_two, d, first):
```

Comment:
Is it possible to change `d` to something more meaningful? 

A good Python style guide from Google - https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming